src/Calibration.cpp

```diff
@@ -1692,6 +1710,7 @@ void Calibration::mergeAllObjectObs() {
 
   // Reinitialize all the 3D object
   for (const auto &it_object : object_3d_) {
+    (void)it_object;
```

Comment:
@rameau-fr it_object is never used. Could you please take a look at this loop?

Reply:
wow ... I do not even understand why we have two loops iterating through the same objects ... Unfortunately, I do not remember the reasoning behind it and it would be tempting to delete one loop but ... I am very unsure! I would recommend to leave it as it is. Since it is "just" the reinitialization of the objects, I do not think it is really harmful.